lib/radar/ical.rb

```diff
@@ -17,8 +17,8 @@ def parse_event(event)
       result = {
         id: event.start.to_s,
         url: event.url.to_s,
-        title: event.summary.force_encoding("utf-8"),
-        description: event.description.force_encoding("utf-8"),
+        title: (event.summary || "").force_encoding("utf-8"),
```

Comment:
Yes, it is nil. `.to_s` will do the same. But I love to write it this way to make it clear. Otherwise I don't know why it is there and remove it :wink: